src/download/exchangeDownloader.ts

```diff
@@ -149,7 +149,7 @@ export async function searchExchange(
   searchString: string
 ): Promise<RestApi[]> {
   return fetch(
-    `${ANYPOINT_BASE_URI}/assets?search=${searchString}&types=rest-api`,
+    `${ANYPOINT_API_URI_V2}/assets?search=${searchString}&types=rest-api`,
```

Comment:
why V2 here?

Reply:
It is just the variable name change...It was V2 always